reference/OpenAPI.json

```diff
@@ -38740,6 +38824,20 @@
           "$ref": "#/components/schemas/CampaignsImportVoucherItem"
         }
       },
+      "CampaignsImportVoucherItem": {
+        "title": "Campaigns Import Voucher Item",
+        "oneOf": [
```

Comment:
Te `oneOf`-y w takich przypadkach robią ogromną robotę, o wiele lepiej to teraz będzie działać 👏